firefox_ios/views/counters.view.lkml

```diff
@@ -215,18 +205,28 @@ view: counters {
     suggest_dimension: metrics_sample.subdivision2
   }
 
+  dimension: label {
+    type: string
+    sql: ${TABLE}.key ;;
+  }
+
+  measure: client_aggregate {
```

Comment:
Isn't this doing the same thing as `total_count`? Will it be clear to end users what `client_aggregate` means in the explore menu?

Reply:
Oh you're right, I totally didn't realize that field was there.